Gems/Terrain/Assets/Shaders/Terrain/TerrainSrg.azsli

```diff
@@ -83,13 +83,19 @@ ShaderResourceGroup TerrainSrg : SRG_Terrain
         float2 m_boundsMax;
     };
 
-    struct MacroMaterialGrid
+    struct MacroMaterialGridInfo
     {
         uint m_resolution; // How many x/y tiles in grid. x & y stored in 16 bits each. Total number of entries in m_macroMaterialData will be x * y
         float m_tileSize; // Size of a tile in meters.
         float2 m_offset; // x/y offset of min x/y corner of grid.
     };
 
+    struct MacroMaterialRefs
+    {
+        uint m_index12;
+        uint m_index34;
```

Comment:
Nit suggestion: using 01, 23 is better to me since they match the index after unpacking.

Reply:
hah, yeah totally agree! indices start at 0 not 1!